ci/test_cpp.sh

```diff
@@ -35,6 +34,8 @@ rapids-mamba-retry install \
 rapids-logger "Check GPU usage"
 nvidia-smi
 
+EXITCODE=0
+trap "EXITCODE=1" ERR
```

Comment:
Can you remove the `TODO` a few lines down?

Reply:
resolved, thanks!